Tasks/AzureFileCopy/Utility.ps1

```diff
@@ -1275,6 +1275,17 @@ function Is-WinRMCustomScriptExtensionExists
     $isExtensionExists
 }
 
+function Get-TargetUriFromFwdLink {
+    param(
+        [string]$fwdLink
+    )   
+
+    Write-Verbose "Trying to get the target uri from the fwdLink: $fwdLink"
+    $targetUri =  $(Invoke-WebRequest $fwdLink).BaseResponse.Uri.AbsoluteUri
```

Comment:
I see few examples where it is mentioned that it is safe to be explicit setting this request.AllowAutoRedirect = true; when we are intending to find the redirect Url.

Reply:
This was not proxy aware earlier. Would it be better to fix that as well?